packages/utils/src/assets/asset-id.ts

```diff
@@ -68,13 +68,13 @@ export function getAssetId(asset: CryptoAsset): CryptoAssetId {
         id: asset.stamp.toString(),
       };
     case 'sip9':
-      return buildSip9AssetId(asset);
+      return createSip9AssetId(asset);
     default:
       assertUnreachable(asset);
   }
 }
 
-function buildSip9AssetId(asset: Sip9Asset): CryptoAssetId {
+export function createSip9AssetId(asset: Sip9Asset): CryptoAssetId {
```

Comment:
I'm making this small change to a util that's only used in `mobile` but the `extension` Playwright tests are failing for Shard 3. 

I guess we need to test everything if we change packages but those tests can be flaky and flag non existent issues